book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  attr_accessor :title, :author
+
+  has_many :rentals
+  has_many :persons, through: :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
```

Comment:
- [ ] Rentals should also be initialized. Please refer to this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md) for more information on how to do that.